src/SFA.DAS.AANHub.Api.UnitTests/Controllers/EmployersControllerTests.cs

```diff
@@ -62,5 +64,47 @@ public EmployersControllerTests()
 
             result.As<BadRequestObjectResult>().StatusCode.Should().Be(StatusCodes.Status400BadRequest);
         }
+
+        [Test, AutoMoqData]
```

Comment:
Add a test for BadRequest response

Reply:
@DevenKShah, did you mean to say a test for `StatusCodes.Status404NotFound`?

Reply:
yes correct